required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -0,0 +1,1731 @@
+%      including \cs{interlinepenalty}, \cs{hsize}, \cs{parindent} and
+%      a number of others, as some of them depend on the font
+%      size. Then the \hook{fntext/para} is run. If one wants to
+%      change the font size, it is probably necessary to reset these
+%      other parameters too, e.g., \cs{parindent}, which can be done
+%      here.
+%
+%    \item[\hook{fntext/para2}]
+%
+%      The configuration point \cs{@footnotetext@cfgpointii} normally
+%      runs the command \cs{@makefntext} or some code that eventually
+%      runs this command, and this then produces the footnote mark (in
+%      front of the footnote text) and the formatted footnote text. In
+%      front of both the mark and the footnote text the hook
+%      \hook{fntext/para2} is executed.  In fact it is not
+%      clear if it is really needed, as most likely all of its
+%      settings could probably go into \hook{fntext/para}, it is just
+%      that historically there have been settings in some classes at
+%      this point in \cs{@makefntext}.
+%
+%    \item[\hook{fntext/begin} \& \hook{fntext/end}]
+%
+%      The footnote text itself is surrounded by the hooks
+%      \hook{fntext/begin} and \hook{fntext/end}. The two hooks are
+%      not paired as they are typically used independently.
+%
+%    \item[\hook{fntext/after}]
+%
+%      At the very end of \cs{footnotetext} we execute the hook
+%      \hook{fntext/after} which is a reversed hook paired with
+%      \hook{fntext/before}. Some packages, e.g., \pkg{linuex}, have
+%      code in that position.
+%
+%    \end{description}
+%
+%
+%
+%
+% \subsubsection{Additional configuration possibilities}
+%
+%    The formatting of the footnote mark in front of the footnote
+%    texts is influenced by the setting of the dimen parameter
+%    \cs{footnotemargin}. By default its value is 1.8em in the current
+%    text font (or \texttt{-}\cs{maxdimen} when the para option is
+%    chosen). The following rules apply:
+%    \begin{itemize}
+%    \item
+%
+%      If it has the value \texttt{-}\cs{maxdimen} then the mark is
+%      generated by \cs{@makefnmark}.
+%
+%    \item
+%
+%      Otherwise, if the value is
+%      negative then the mark is placed into an \cs{llap} left aligned
+%      in a box of size \texttt{-}\cs{footnotemargin}.
+%
+%    \item
+%
+%      If the value is zero an \cs{llap} is used without an inner box.
+%
+%    \item
+%
+%      If the value is greater zero (but less than \cs{maxdimen}) the
+%      mark is placed right aligned into a box of size
+%      \cs{footnotemargin}.
+%
+%    \item
+%
+%      The value \cs{maxdimen} is used as a marker to indicate that
+%      no value was given and that the default should be used,
+%      i.e. 1.8em or \texttt{-}\cs{maxdimen} depending on the chosen
+%      option.
+%    \end{itemize}
+%
+%
+% \section{Tagging support}
+%
+%
+%
+%
+%
+%
+% \StopEventually{\setlength\IndexMin{200pt}  \PrintIndex  }
```

Comment:
`\MaybeStop`?

Reply:
upps, wonder where that came from ... not in footmisc.dtx